src/Symfony/Component/Security/Http/Tests/EventListener/SessionLogoutListenerTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Http\Tests\EventListener;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\Security\Http\Event\LogoutEvent;
+use Symfony\Component\Security\Http\EventListener\SessionLogoutListener;
+
+class SessionLogoutListenerTest extends TestCase
```

Comment:
```suggestion
final class SessionLogoutListenerTest extends TestCase
```

Reply:
The test class don't need to be `final`. Here, be consistent with others.